dired-sidebar.el

```diff
@@ -553,12 +558,15 @@ With universal argument, use current directory."
   (interactive)
   (if (dired-sidebar-showing-sidebar-p)
       (dired-sidebar-hide-sidebar)
-    (let* ((file-to-show (dired-sidebar-get-file-to-show))
+    (let* ((old-buf (dired-sidebar-buffer (selected-frame)))
```

Comment:
Nit: Can you change this from old-buf to old-buffer?

Reply:
Got it